servers/rendering/rendering_server_default.h

```diff
@@ -74,7 +74,7 @@ class RenderingServerDefault : public RenderingServer {
 	uint64_t print_frame_profile_ticks_from = 0;
 	uint32_t print_frame_profile_frame_count = 0;
 
-	mutable CommandQueueMT command_queue;
+	mutable CommandQueueMT command_queue(true);
```

Comment:
I guess this needs to be:
```suggestion
	mutable CommandQueueMT command_queue = CommandQueueMT(true);
```